packages/sync/src/modules/class-term-relationships.php

```diff
@@ -131,7 +131,7 @@ public function enqueue_full_sync_actions( $config, $max_items_to_enqueue, $last
 
 		// We need to do this extra check in case $max_items_to_enqueue * $term_relationships_full_sync_item_size == relationships objects left.
 		$count = $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->term_relationships WHERE ( object_id = %d AND term_taxonomy_id < %d ) OR ( object_id < %d ) ORDER BY object_id DESC, term_taxonomy_id DESC LIMIT %d", $last_object_enqueued['object_id'], $last_object_enqueued['term_taxonomy_id'], $last_object_enqueued['object_id'], 1 ) );
-		if ( intval( $count ) === 0 ) {
+		if ( (int) $count === 0 ) {
```

Comment:
Since we're touching that line, let's use Yoda conditions here:

```suggestion
		if ( 0 === (int) $count ) {
```